GS_example.py

```diff
@@ -36,21 +40,27 @@
           np.arange(-params.matrix_size / 2, params.matrix_size / 2) * params.pixel] for y in
          np.arange(-params.matrix_size / 2, params.matrix_size / 2) * params.pixel])
 
+    # Prepare optimizer
     GS = GerchbergSaxton(params)
 
+    # Run optimizer
     res = GS.optimize(LightField(amp, phase), LightField(target, phase), 5)
 
+    # Plot the result - optimized phase map
     plotter = GeneratePropagationPlot(res[0], output_type=GeneratePropagationPlot.PLOT_PHASE)
     plotter.save_output_as_figure("outs/structure.png")
     plotter.show()
 
+    # Plot the input amplitude
     plotter = GeneratePropagationPlot(res[0], output_type=GeneratePropagationPlot.PLOT_ABS)
     plotter.save_output_as_figure("outs/input_field.png")
-    
+
+    # Plot the result - output amplitude
     plotter = GeneratePropagationPlot(res[1], output_type=GeneratePropagationPlot.PLOT_ABS)
     plotter.save_output_as_figure("outs/result.png")
     plotter.show()
 
+    # Plot the target amplitude
     plotter = GeneratePropagationPlot(LightField(target, phase), output_type=GeneratePropagationPlot.PLOT_ABS)
```

Comment:
Can't we "static import" constants from `GeneratePropagationPlot`? I think it will make this more readable

Reply:
Good idea